File: Basic/Python/Day-10/Stack.py
stack = []
stack.append(10)
stack.append(20)
stack.append(30)
print(f"Elements in stacks: {stack}")
print(f"{stack[-1]} to see the top element before deletion by using peek.") #it act as peak to see the top end element of stack..
print(f"{stack.pop()} is deleted from the stack.") #pop is used to remove the top end element of stack..and it returns that value..
print(f"{stack[-1]} to see the top element after deletion by using peek.")
print(f"Remaining elements in stacks: {stack}")

File: Basic/Java/Day-10/Stack.java
import java.util.ArrayDeque;
import java.util.Deque;

public class Stack{
    public static void main(String[] args) {
        Deque<Integer> stack = new ArrayDeque<>();
        stack.push(12);
        stack.push(13);
        stack.push(14);
        stack.push(16);
        System.out.println(stack);
        System.out.println("See top element before deletion : " + stack.peek());
        System.out.println("Popped top element : " + stack.pop());
        System.out.println("See top element after deletion : " + stack.peek());
        System.out.println("Check stack is empty or not : " + stack.isEmpty());
        System.out.println("Check size of stack : " + stack.size());
        System.out.println(stack);
    }
}